src/jukebox/components/battery_monitor/batt_mon_i2c_ina219/__init__.py

```diff
@@ -0,0 +1,94 @@
+# MIT License
+#
+# Copyright (c) 2021 Arne Pagel
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+# Contributing author(s):
+# - Arne Pagel
+
+import logging
+import jukebox.plugs as plugs
+import jukebox.cfghandler
+from ina219 import INA219
+from ina219 import DeviceRangeError
+from components.battery_monitor import BatteryMonitorBase
+
+logger = logging.getLogger('jb.battmon')
+
+batt_mon = None
+
+
+class battmon_ina219(BatteryMonitorBase.BattmonBase):
+    '''Battery Monitor based on a INA219
```

Comment:
Could you extract the documentation in this path `/documentation/builders/components/power/batterymonitor.md`

Depending how much the docs differ to the existing battery monitor you could either extend the existing doc file or add a separate file in that path.

The existing file lacks some info how the plug-in is „activated“ anyway.

Reply:
Sure. I will improve the documentation.